### PR TITLE
Release-EUI-2439-3787-3843

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-2439
https://tools.hmcts.net/jira/browse/EUI-3787
https://tools.hmcts.net/jira/browse/EUI-3843

### Change description ###
Release for the above mentioned tickets

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Mo-Lala84 — DISMISSED** [review]
approved

File: src/app/components/cookie-policy/cookie-policy-component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CookiePolicyComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```
Comment (kuda1992) [inline src/app/components/cookie-policy/cookie-policy-component.spec.ts:30]: What is the point of these tests they do nothing . 

`it('should create'')`
`it('should be created by angular')`

**Review by kuda1992 — CHANGES_REQUESTED** [review]
Just a few changes

File: src/app/components/cookie-policy/cookie-policy-component.spec.ts (hunk 1 above)
Comment (RiteshHMCTS) [reply src/app/components/cookie-policy/cookie-policy-component.spec.ts]: removed these tests.

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by kuda1992 — APPROVED** [review]

**Pull request merged**